src/models/chainspec.rs

```diff
@@ -151,7 +147,11 @@ pub struct ConsensusParams {
     pub eip1559_block: Option<BlockNumber>,
 }
 
-#[derive(Debug, Serialize, Deserialize, PartialEq)]
+pub fn switch_is_active(switch: Option<BlockNumber>, block_number: BlockNumber) -> bool {
+    block_number >= switch.unwrap_or(BlockNumber(u64::MAX))
+}
```

Comment:
```suggestion
pub fn is_block_num_after_fork(fork_block_num: Option<BlockNumber>, block_number: BlockNumber) -> bool {
    block_number >= fork_block_num.unwrap_or(BlockNumber(u64::MAX))
}
```

Reply:
The original meaning signals the purpose: that the EIP in question is active for this block.